pyflakes/test/test_api.py

```diff
@@ -495,11 +487,10 @@ def test_eofSyntaxErrorWithTab(self):
 
             self.assertHasErrors(
                 sourcePath,
-                ["""\
```

Comment:
I prefer the escaped newline so the full error message is all on the left -- not a big deal though